src/studio/src/designer/backend/Languages/ini/nb.ini

```diff
@@ -394,8 +397,9 @@ enum = Gyldige verdier
 field = Felt
 fields = Felter
 go_to_type = Gå til type
-group = Gruppe
+group = allOf/anyOf/oneOf
 import = Importer
+inline_information = I dag støtter vi bare referanser som subskjema når man kombinerer skjema (allOf, anyOf og oneOf). Subskjema som ikke er referanser vil bli vist i det grå feltet over, men det vil ikke være mulig å editere de i editoren her enda.
```

Comment:
`inline_information` could it be a bit more specific? Maybe `sub_schema_disclaimer`?